src/Cake/project.json

```diff
@@ -1,8 +1,9 @@
 ﻿{
-  "version": "0.16.1-*",
+  "version": "0.16.2-*",
```

Comment:
@devlead the reason that I ask is that in the last PR you did for 0.16.1, you updated all the files 😄